general/envsetup/envsetup.go

```diff
@@ -12,12 +9,7 @@ const (
 	registrationPageURL = "https://jfrog.com/start-free/cli/"
 )
 
-func RunEnvSetupCmd() error {
-	fmt.Println()
-	fmt.Println()
-	fmt.Println(coreutils.PrintTitle("Thank you for installing JFrog CLI! 🐸"))
-	fmt.Println(coreutils.PrintTitle("We'll now set up a FREE JFrog environment in the cloud for you, and configure your local machine to use it."))
-	fmt.Println("Your environment will be ready in less than a minute.")
-	setupCmd := envsetup.NewEnvSetupCommand(registrationPageURL)
```

Comment:
The above line should not have been removed. They were moved to *jfrog-cli* from *jfrog-cli-core* recenetly. See my corresponding comment on your dependent *jfrog-cli-core* PR.